libs/wingsdk/patches/jsii+1.74.0.patch

```diff
@@ -1,3 +1,19 @@
+diff --git a/node_modules/jsii/lib/assembler.js b/node_modules/jsii/lib/assembler.js
+index fe972cb..caa088f 100644
+--- a/node_modules/jsii/lib/assembler.js
++++ b/node_modules/jsii/lib/assembler.js
+@@ -1557,9 +1557,8 @@ class Assembler {
+                 this._diagnostics.push(jsii_diagnostic_1.JsiiDiagnostic.JSII_1002_UNSPECIFIED_PROMISE.create(declaration));
+                 return { type: spec.CANONICAL_ANY };
+             }
+-            return {
+-                type: this._typeReference(typeRef.typeArguments[0], declaration, purpose),
+-            };
++            // (very very hacky) allow optionals to be returned from async methods
++            return this._optionalValue(typeRef.typeArguments[0], declaration, purpose);
```

Comment:
it's hacky I guess because I'm not 100% sure why jsii disallows this behavior / if this limitation is intentional or not. I can definitely open an issue in their repo to see if it can be upstreamed